advisories/hackage/crypton/HSEC-0000-0000.md

```diff
@@ -0,0 +1,36 @@
+```toml
+[advisory]
+id = "HSEC-0000-0000"
+keywords = ["crypto"]
+[[references]]
+type = "FIX"
+url = "https://github.com/kazu-yamamoto/crypton-certificate/commit/e353d450c381c9d6b903c4257927e0c89c97acb1"
+
+[[affected]]
+package = "cryptonite"
+cvss = "CVSS:4.0/AV:L/AC:H/AT:P/PR:L/UI:P/VC:L/VI:N/VA:N/SC:N/SI:N/SA:N"
```

Comment:
CVSS version 4 is not yet supported, would you mind using version 3.1 instead?

Reply:
+1